src/styles/blocks/header.scss

```diff
@@ -0,0 +1,53 @@
+.header {
+  display: flex;
+  align-items: center;
+  justify-content: space-between;
+  padding: 0 50px;
+  background-color: #fff;
+  box-shadow: 0 2px 4px 0 #0000000d;
+}
+
+.logo {
+  display: flex;
+}
+
+.nav__list {
+  display: flex;
+  list-style: none;
+  margin: 0;
+  padding: 0;
+  justify-content: center;
+  align-items: center;
+
+  & .nav__item:not(:last-child) {
+    margin-right: 20px;
+  }
+}
+
+.nav__link {
+  text-transform: uppercase;
+  display: flex;
+  color: #060b35;
+  text-decoration: none;
+  font-size: 12px;
+  line-height: 60px;
+  text-align: left;
+  align-items: center;
+  position: relative;
+}
+
+.is-active,
+.nav__link:hover {
```

Comment:
The selector `.is-active` should be more specific to avoid potential conflicts with other elements that might use the same class name. Consider using a more specific selector like `.nav__link.is-active`.